File: 01 Python Programming Fundamentals/Lesson 4 Loops/Loops.cpp
#include <iostream>
using namespace std;
/*
Created by Jonatan Gomez Garcua on 05-01-2024
*/


int main() {
    //Mainly intended for searches
    int count = 0;
    bool found = false;
    while(count < 5 && !found){
        cout << "Round number while loop " << count << endl;
        
        //Exit condition
        if (count == 3){ 
            found = true; 
            cout << "3 is found " << endl;
        }

        count ++;
    }

    //Mainly intended for go over
    for(int i = 0; i < 5; i++){
        cout << "Round number for loop " << i << endl;
    }

        count = 0;
        found = false;
    do{
        cout << "Round number do-while loop " << count << endl;
        //Exit condition
        if (count == 3){ 
            found = true; 
            cout << "3 is found " << endl;
        }

        count ++;
    } while (count < 5 && !found);

    return 0;
}

File: 00 Python Programming Fundamentals/Lesson 4 Loops/Loops.py
#!/bin/python3
''' 
Created by Jonatan Gomez Garcia on 05-01-2024
'''

count = 0
found = False
#Mainly intended for searches
while count < 5 and not found:
    print(f"Round number while loop {count}")
    if count == 3:
        found = True
        print('3 is found')
    count+=1
    
#Mainly intended for go over
for i in range(6):
    print(f"Round number for loop {i}")
    
world = 'Hello World!'

#Mainly intended for go over
for character in world:
    print(character)
    
